internal/payloads/error_payload.go

```diff
@@ -199,3 +201,13 @@ func NewGCPError(ctx context.Context, message string, err error) *ResponseError
 	message = fmt.Sprintf("Google API error: %s", message)
 	return NewResponseError(ctx, http.StatusInternalServerError, message, err)
 }
+
+func HandleGetSourcesAuthenticationError(ctx context.Context, err error) *ResponseError {
+	if errors.Is(err, clients.ErrNotFound) {
+		return NewNotFoundError(ctx, "unable to get authentication for sources", err)
+	}
+	if errors.Is(err, clients.ErrBadRequest) {
+		return NewResponseError(ctx, http.StatusBadRequest, "unable to get authentication from sources", err)
+	}
+	return NewClientError(ctx, err)
+}
```

Comment:
This is okay, but can you do it in `NewClientError` function instead? We already do this for user messages, we can do the similar check too. Then there is no need to change all the services:

```
func NewClientError(ctx context.Context, err error) *ResponseError {
	if response := findUserResponse(ctx, "Client error", err); response != nil {
		return response
	}
	log.Ctx(ctx).Error().Msgf("Unknown client error: %s", err)
	return NewResponseError(ctx, 500, "backend client error", err)
}
```